fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchAllTest.java

```diff
@@ -47,14 +49,15 @@
     public void testCreatePatient() throws Exception {
         WebTarget target = getWebTarget();
 
-        // Build a new Patient and then call the 'create' API.
+        // Build a new Patient with 2 tags and then call the 'create' API.
         Patient patient = TestUtil.readLocalResource("Patient_JohnDoe.json");
 
         Coding security = Coding.builder().system(uri("http://ibm.com/fhir/security")).code(code("security")).build();
         Coding tag = Coding.builder().system(uri("http://ibm.com/fhir/tag")).code(code("tag")).build();
+        Coding tag2 = Coding.builder().system(uri("http://ibm.com/fhir/tag")).code(code("tag2")).build();
 
-        patient =
-                patient.toBuilder().meta(Meta.builder().security(security).tag(tag).profile(Canonical.of("http://ibm.com/fhir/profile/Profile")).build()).build();
+        patient = patient.toBuilder().meta(Meta.builder().security(security)
+                        .tag(tag).tag(tag2).profile(Canonical.of("http://ibm.com/fhir/profile/Profile")).build()).build();
```

Comment:
i think its more readable if you follow convension used in other places:
example:
```
patient.toBuilder().meta(Meta.builder().security(security)
                        .tag(tag).tag(tag2).profile(Canonical.of("http://ibm.com/fhir/profile/Profile")).build()).build();
Commenting on lines +59 to +60
Beta Send feedback
 

 
i think its more readable if you follow convension used in other places:
example:
```
patient.toBuilder()
            .meta(Meta.builder()
                               .security(security)
                               .tag(tag)
                               .tag(tag2)
                               .profile(Canonical.of("http://ibm.com/fhir/profile/Profile"))
                               .build())
            .build();
```

Reply:
sounds reasonable for me! :)